app.rb

```diff
@@ -65,18 +58,22 @@ def reset_game
       session[:guesses] = nil
       session[:letters] = nil
     end
+
+    def format(wrong_guesses)
+      wrong_guesses.map {|guess| guess.capitalize}.join(" ")
+    end
   end
 
   get "/" do
     @image = ImageLibrary.new.current_image(guesses[:wrong_guesses])
     @secret_word = WordFormatter.new(rules).format(secret_word)
-    @letters = letters.join(" ")
+    @wrong_guesses = format(guesses[:wrong_guesses])
```

Comment:
Well done for keeping the logic of capitalising the letters away from the front end templates. But couldn't the formatting also be done by the `WordFormatter`?

Reply:
But this `format()` function is formatting the wrong guesses, not the secret word?

Reply:
You could rename the WordFormatter? Look into the presenter pattern. That is maybe what you're using here (or could use more explicitly)